core/src/test/kotlin/org/stellar/anchor/auth/AuthHelperTest.kt

```diff
@@ -4,38 +4,34 @@ import io.mockk.*
 import java.util.*
 import kotlin.test.assertEquals
 import kotlin.test.assertNull
-import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.parallel.Execution
+import org.junit.jupiter.api.parallel.ExecutionMode.SAME_THREAD
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.EnumSource
-import org.stellar.anchor.auth.ApiAuthJwt.CallbackAuthJwt
-import org.stellar.anchor.auth.ApiAuthJwt.CustodyAuthJwt
-import org.stellar.anchor.auth.ApiAuthJwt.PlatformAuthJwt
+import org.stellar.anchor.auth.ApiAuthJwt.*
 import org.stellar.anchor.auth.AuthType.*
 import org.stellar.anchor.util.AuthHeader
 
+@Execution(SAME_THREAD)
+@Order(86)
 class AuthHelperTest {
   companion object {
     const val JWT_EXPIRATION_MILLISECONDS: Long = 90000
   }
 
-  @AfterEach
-  fun teardown() {
-    clearAllMocks()
-    unmockkAll()
-  }
-
   @ParameterizedTest
   @EnumSource(AuthType::class)
   fun `test AuthHeader creation based on the AuthType`(authType: AuthType) {
     when (authType) {
       JWT -> {
         // Mock calendar to guarantee the jwt token format
+        mockkStatic(Calendar::class)
```

Comment:
Can we create a ticket to remove all static mocks from the tests?

Reply:
https://stellarorg.atlassian.net/browse/ANCHOR-488